### PR TITLE
Bug/ Remove log from point share, fix warnings from choose child child modal and parent dashboard.  

### PR DESCRIPTION
**Pull request by rhiannonstanford**
Removed unnecessary log from PointShare to clear console.  

Resolved warnings and errors from Choose Child Modal and Parent Dashboard.

Screen Shot 2021-11-08 at 8 26 05 PM
Screen Shot 2021-11-08 at 8 44 38 PM
Screen Shot 2021-11-08 at 8 34 09 PM
Screen Shot 2021-11-08 at 8 23 05 PM

### CODE REVIEW

**Review by rhiannonstanford — COMMENTED** [review]
Self review.

**Review by ethankarp — APPROVED** [review]
Nice job making these fixes and improvements!

File: src/components/pages/NewParentDashboard/ChooseChildModal.js [hunk 1]
```diff
@@ -69,9 +72,12 @@ export default function ChooseChildModal(props) {
                       cursor: 'pointer',
                       justifyContent: 'center',
                     }}
-                    onClick={evt => handleCharacterClick(evt, ID)}
+                    onClick={evt => {
```
Comment (Julianisrel) [inline src/components/pages/NewParentDashboard/ChooseChildModal.js:75]: Nice job!

**Review by Julianisrel — APPROVED** [review]

**Review by milklot — APPROVED** [review]
Looks much cleaner , good job!

**Pull request merged**